README.md

```diff
@@ -1,244 +1,244 @@
 
-  - #### Sanic
+  - #### سنیک
 
   - #### AIOHTTP
 
-  - #### Bottle
+  - #### باتل
 
-  - #### Dash
+  - #### دش
 
   - [List Of All Python Backend Web Frameworks](https://wiki.python.org/moin/WebFrameworks)
 
-- ### Data Science
+- ### علم داده
 
-- ### Machine Learning
+- ### یادگیری ماشین
 
-- ### Deep Learning
+- ### یادگیری عمیق
 
-- ### DevOps
+- ### دوآپس
 
-- ### Hacking
+- ### هک و امنیت
 
-- ### Algorithmic Trading
+- ### مبادله الگوریتمیک
 
-## Advanced Topics
+## مباحث پیشرفته
 
-- ### Databases
+- ### پایگاه داده ها
   
-  - #### General
+  - #### عمومی
 
-  - #### PostgreSQL
+  - #### پستگرس کیو ال
   
-    - **Video**
+    - **ویدیو**
 
       - [Database concepts by Alireza Morteza Amini](https://maktabkhooneh.org/course/%D9%BE%D8%A7%DB%8C%DA%AF%D8%A7%D9%87-%D8%AF%D8%A7%D8%AF%D9%87-mk214/)
       - [PostgreSQL Tutorial by Alireza Ayinmehr](https://www.youtube.com/playlist?list=PLdUn5H7OTUk1PSzkW_1KmEX_wASLw7X_M)
 
-  - #### MongoDB
+  - #### مونگو دی بی
 
-  - #### Redis
+  - #### ردیس
 
-    - **Video**
+    - **ویدیو**
       - [Redis in one hour](https://www.youtube.com/watch?v=J_4gThmGT9o)
 
-  - #### MemCached
+  - #### مم کشد
   
-- ### Clean Code
+- ### کد تمیز
 
-- ### Clean Architecture
+- ### معماری تمیز
 
 - ### Caching
 
 - ### Testing
 
-- ### Container Platforms
+- ### نرم افزار های کانتینر
 
-  - #### Docker
+  - #### داکر
   
-    - **Book**
+    - **کتاب**
 
-    - **Video**
+    - **ویدیو**
       - [Docker Tutorial by Mehran Tarif](https://www.youtube.com/watch?v=j4mYb5ZuH3Q&list=PLAt10Vana3Ye_xFzq-whnnOhPsHQe1NDX)
       - [Docker Tutorial by Ahmad Rafiee](https://dockerme.ir/category/video/)
       - [Docker in Action by Ahmad Tahani](https://faranesh.com/programming/14863-docker-in-action)
       - [Docker Course by Jadi](https://www.youtube.com/watch?v=_jKNnHROiC0)
 
-  - #### Kubernetes
+  - #### کوبرنتز
 
-- ### Programming Paradigms
+- ### الگو های برنامه نویسی
 
-- ### Architectural Patterns
+- ### الگو های معماری
 
-- ### Design Principles
+- ### اصول طراحی
   
-- ### Design Patterns
+- ### الگو های طراحی
 
 - ### Message Brokers
 
 - ### Web Servers
 
-- ### Availability and Reliability
+- ### دسترسی و پایداری
 
-- ### Distributed Systems
+- ### سیستم های توزیع شده
 
-- ### Reactive Systems
+- ### سیستم های واکنش پذیر
 
 - ### Refactoring
 
-- ### Security
+- ### امنیت
 
-- ### Monitoring
+- ### نظارت
 
-- ### Soft Skill
+- ### مهارت های نرم
 
 - ### Public Cloud
 
-- ### Where to Go Next?
+- ### بعد از این چه کاری انجام دهیم؟
 
-## Contribution
+## مشارکت
 
-Feel free to contribute and add more resources.
+با خیال راحت مشارکت کنید و منابع بیشتری اضافه کنید.
 
-First time contributing to a repository? For more information about the steps and guides, check out [this repository](https://github.com/firstcontributions/first-contributions). ([Also available in Persian](https://github.com/firstcontributions/first-contributions/blob/master/translations/README.fa.md))
```

Comment:
Hey there! I think it's better to keep the link to the [Persian version of contribution guide lines](https://github.com/firstcontributions/first-contributions/blob/master/translations/README.fa.md) since this repository is mostly reviewed by the Persian community.

Best regards.

Reply:
Thank you for your helpful review.
Can I change the link completely to the Persian version? This way we could remove the  '([Also available in Persian])' from this roadmap and the English roadmap.

Wish you the best.

Reply:
Yes I think that makes more sense than this one. Cheers!

Reply:
Perfect, Will do the changes now ! 
Best wishes.